src/managers/FirmwareCompilationManager.js

```diff
@@ -117,7 +118,7 @@ class FirmwareCompilationManager {
 
     const errors = [];
     makeProcess.stderr.on('data', (data: string) => {
-      logger.error(`${data}`);
+      logger.error({ errordata: data }, 'Error from MakeProcess');
```

Comment:
just `data`, we should already know that its errorData since its in logger.error